src/appleseed.shared/application/application.cpp

```diff
@@ -137,6 +137,40 @@ const char* Application::get_root_path()
     return root_path_buffer;
 }
 
+const char* Application::get_user_settings_path()
+{
+    static char user_settings_buffer[FOUNDATION_MAX_PATH_LENGTH + 1];
+    static bool user_settings_initialized = false;
+
+    if (!user_settings_initialized)
+    {
+// Windows.
+#if defined _WIN32
+
+        return 0;
+
+// OS X.
+#elif defined __APPLE__
+
+        return 0;
+
```

Comment:
For consistency, you could've added `// Unix.` comment here (or something like "Other Unix-like" or whatever), but that's not really that important.